src/wallet/rpcwallet.cpp

```diff
@@ -1110,7 +1110,7 @@ UniValue addmultisigaddress(const JSONRPCRequest& request)
     if (request.fHelp || request.params.size() < 2 || request.params.size() > 3)
     {
         std::string msg = "addmultisigaddress nrequired [\"key\",...] ( \"account\" )\n"
-            "\nAdd a nrequired-to-sign multisignature address to the wallet.\n"
+            "\nAdd a nrequired-to-sign multisignature address to the wallet. Requires a new wallet backup.\n"
```

Comment:
"nrequired-to-sign multisignature"

Only comment, I'm assuming this isn't a typo, and is a reference to the number of required signatures needed for the multisig address?

Reply:
Correct :) But note that that text is not changed in this PR, it was there before

Reply:
Ok sweet, yeah I saw that it wasn't a change from this commit, but wasn't sure so figured I'd ask. Thanks! Is it typically frowned upon to comment on things not directly changed by the PR/Commit/Issue in question?

Reply:
No, if something looks wrong then feel free to point it out :) I was just clarifying

Reply:
Awesome, thanks again.